src/main/java/gift/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package gift.exception;
+
+import jakarta.validation.ConstraintViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<Map<String, String>> handleValidation(MethodArgumentNotValidException ex) {
+        Map<String, String> errors = new HashMap<>();
+        ex.getBindingResult().getFieldErrors()
+                .forEach(err -> errors.put(err.getField(), err.getDefaultMessage()));
+        return ResponseEntity.badRequest().body(errors);
+    }
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<String> handleIllegalArg(IllegalArgumentException ex) {
```

Comment:
```suggestion
    public ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException ex) {
```
Exception명을 풀네임으로 쓰는 건 어떨까요? 어떤 exception을 다루는 지 알기 쉬울 것 같아요!

Reply:
handleIllegalArgumentException으로 변경했습니다 !